src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -270,6 +270,7 @@
     <Compile Include="$(CommonPath)Interop\Windows\OleAut32\Interop.SysAllocStringByteLen.cs">
       <Link>Common\Interop\Windows\OleAut32\Interop.SysAllocStringByteLen.cs</Link>
     </Compile>
+    <Compile Include="ILLinker\ILLink.Suppressions.cs" />
```

Comment:
NIT: Consider adding this on the eng\illink.targets file so that we automatically add this to the compile in case it exists. This is what we do in other baseline cases for example: [https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.ApiCompat/build/Microsoft.DotNet.ApiCompat.targets#L31-L32](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.ApiCompat/build/Microsoft.DotNet.ApiCompat.targets#L31-L32)

That way adding a new supression doesn't mean you have to go author the .csproj which might be harder for folks to do if they are not familiar with msbuild.

Reply:
I think this is more than a `NIT`. 😄 We should pick it up automatically.